ci-operator/config/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main.yaml

```diff
@@ -19,6 +19,7 @@ images:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
   to: ibmcloud-machine-controllers
 promotion:
+  disabled: true
```

Comment:
Why is promotion being disabled?

Reply:
I think we cannot have image promotion from two places since we have adding configuration in `machine-api-provider-ibmcloud` directory we need to disable it from `cluster-api-provider-ibmcloud` directory, Otherwise `make jobs` will result infollowing error
```docker run --platform linux/amd64 --rm -v "/Users/karthikkn/karthik-openshift-workspace/release/ci-operator/config:/ci-operator/config:z" -v "/Users/karthikkn/karthik-openshift-workspace/release/ci-operator/step-registry:/ci-operator/step-registry:z" registry.ci.openshift.org/ci/ci-operator-checkconfig:latest --config-dir /ci-operator/config --registry /ci-operator/step-registry
time="2022-07-25T09:09:46Z" level=error error="output tag ocp/4.12:ibmcloud-machine-controllers is promoted from more than one place: openshift/cluster-api-provider-ibmcloud@main, openshift/machine-api-provider-ibmcloud@main"
time="2022-07-25T09:09:46Z" level=fatal msg="error validating configuration files"```

Reply:
@Karthik-K-N You should then remove the image from the other configuration. The images from this configuration should be built and promoted in the 4.12 steams.

Reply:
so ya, `disabled: true` will do that right

Reply:
No, I mean that here the promotion should be enabled.